x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.09.20.ebuild

```diff
@@ -1,27 +1,22 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit gnome2-utils
+inherit xdg
 
-DESCRIPTION="An icon theme for Linux from the Numix project"
+DESCRIPTION="icon theme from the Numix project"
```

Comment:
(this and multiple other occurrences) Let's start `DESCRIPTION` with capital letters in general.

Reply:
done